components/internal_server/requirements/requirements.in

```diff
@@ -0,0 +1,4 @@
+-r ../../../requirements/bottle.in
+-r ../../../requirements/lxml.in
+gevent==21.12.0
+pymongo==4.1.1
```

Comment:
The external server needs gevent and pymongo too, maybe put them in ../../../requirements/gevent.in and ../../../requirements/pymongo.in, respectively?

Reply:
You use these requirements files to ensure identical versions?

Reply:
Yes, and it's an attempt to reduce duplication among the requirements files.